### ISSUE TITLE
extend Wikipedia page

### ISSUE THREAD

**Issue by Erkan-Yilmaz** [Erkan-Yilmaz/Gridcoin-tasks#12]
current Wikipedia page [here](https://en.wikipedia.org/wiki/Gridcoin)

see https://cryptocointalk.com/topic/42423-wikipediaorg-article-development/
from [hangout 1](https://steemit.com/beyondbitcoin/@cm-steem/gridcoin-hangout-001-rsvp-and-suggest-topics)

**Comment by grctest** [comment]
Are we allowed to use this page as a directory? Like links to websites which support GRC (shops, stats services, links to our team's BOINC stats, important steemit threads, exchanges, etc)?

Would be cool to add a wee section about the hangouts too.

**Comment by Erkan-Yilmaz** [comment]
I'm a long-time editor there:

> Are we allowed to use this page as a directory?
> Like links to websites which support GRC (shops, stats services, links to our team's BOINC stats, important steemit threads, exchanges, etc)?

No, better to appear there with few links or even only 1 to your site which has all that info
(but links as reference to support what one writes at the WP article is good).
Having "fights"/discussions about such things on the talk page there does not look professional.

Wikipedia is in the top5 popular websites worldwide, so having a good profile there is very valuable

**Comment by Peppernrino** [comment]
added currency infobox, release version and date, fixed some dates, removed some unused values from software infobox, added the logo. :)

looking much better now!

**Comment by Peppernrino** [comment]
also updated https://de.wikipedia.org/wiki/Gridcoin :D

**Comment by fkinglag** [comment]
Added plenty of new things and more to come. I think we could try to avoid adding too many things that make it look like a directory. I know that having too many external links can be distracting and also not look too good upon the article as a whole. We should try to have an aim to avoid cluttering up the wikipedia page, too.

**Comment by Erkan-Yilmaz** [comment]
Feb 8: [Removing Gridcoin_logo.png, it has been deleted from Commons by Yann because: Copyright violation, see c:Commons:Licensing.](https://en.wikipedia.org/w/index.php?title=Gridcoin&type=revision&diff=764445962&oldid=764384484)

https://commons.wikimedia.org/wiki/File:Gridcoin_logo.png

**Comment by Peppernrino** [comment]
yes sorry. i didn't have the artist's name. i think barton dug it up for me... it's somewhere in the logs. apologies. their copyright system is very confusing, and it said i had like a week to change it, and then that dude deleted it by the end of the afternoon. :( i'll get it done again asap.

**Comment by Peppernrino** [comment]
fixed.

**Comment by MagixInTheAir** [comment]
Does it look good now or should it be extended even more ?
